tests/Backend/Mixed/SharingTest.php

```diff
@@ -1200,6 +1231,274 @@ function (array $column) {
+
+        $mdList = $this->prepareTestMetadata($bucketId, $aliasAliasTableId, $columnId);
+
+        $response = $this->_client2->getSharedBucketDetail(
+            $project['id'],
+            $bucketId,
+            [
+                'include' => 'columns,tableMetadata,columnMetadata'
+            ]
+        );
+
+        $this->assertSame($bucketId, $response['id']);
+
+        $tables = array_values(array_filter($response['tables'], function ($table) use ($aliasAliasTableId) {
+            return $table['id'] === $aliasAliasTableId;
+        }));
+
+        $this->assertNotEmpty($tables[0]['columnMetadata']['id']);
+        $this->assertEquals(
+            $mdList,
+            $tables[0]['columnMetadata']['id']
+        );
+
+        $this->assertArrayHasKey('metadata', $tables[0]);
+        $this->assertArrayHasKey('columns', $tables[0]);
+        
+        $this->assertSame(5, count($tables[0]['columns']));
+        foreach ($tables[0]['columns'] as $column) {
+            $this->assertTrue(in_array($column, $tableColumns));
+        }
+
+        $tableMetadata = reset($tables[0]['metadata']);
+
+        $this->assertArrayHasKey('id', $tableMetadata);
+        $this->assertArrayHasKey('key', $tableMetadata);
+        $this->assertArrayHasKey('value', $tableMetadata);
+        $this->assertArrayHasKey('provider', $tableMetadata);
+        $this->assertArrayHasKey('timestamp', $tableMetadata);
+
+        $this->assertEquals('test', $tableMetadata['provider']);
+        $this->assertEquals('test.metadata.key', $tableMetadata['key']);
+        $this->assertEquals('test.metadata.value', $tableMetadata['value']);
+    }
+
+    private function prepareTestMetadata($bucketId, $tableId, $columnId)
+    {
+        $metadataClient = new Metadata($this->_client);
+        $metadataClient->postBucketMetadata(
+            $bucketId,
+            'test',
+            [
+                [
+                    'key' => 'test.metadata.key',
+                    'value' => 'test.metadata.value',
+                ],
+            ]
+        );
+
+        $metadataClient->postTableMetadata(
+            $tableId,
+            'test',
+            [
+                [
+                    'key' => 'test.metadata.key',
+                    'value' => 'test.metadata.value',
+                ],
+            ]
+        );
+
+        $metadataClient->postColumnMetadata(
+            $columnId,
+            'test',
+            [
+                [
+                    'key' => 'test.metadata.key',
+                    'value' => 'test.metadata.value',
+                ],
+            ]
+        );
+
+        return $metadataClient->listColumnMetadata($columnId);
+    }
+
+    /**
+     * @dataProvider sharingBackendData
+     * @throws ClientException
+     */
+    public function testGetSharedBucketDetailForNonExistBucket($backend)
+    {
+        $response = $this->_client->verifyToken();
+        $this->assertArrayHasKey('owner', $response);
```

Comment:
na tohle bych dlabal - normálně jen $project = $response['owner']['id']. 

IMHO by se nemělo assertovat nic co se přímo netestuje. Kdyžtak se to vysype normálně na Notice, že neexistuje klíč v poli.

Reply:
Ale tady už to samozřejmně nech

Reply:
Jo ok, tak som to nechal tak. Vsimol som si este parameter `$backend` v deklarovani metody tak ten som zmazal.